content/rust/concepts/strings/strings.md

```diff
@@ -0,0 +1,146 @@
+---
```

Comment:
The metadata is not formatted correctly. It should be formatted like this, the line spaces, characters should be strictly like the below example - 

---
Title: 'The Title'
Description: 'A brief description'
Subjects:
  - 'A subject name'
  - 'A second subject name'
  - 'An nth subject name'
Tags:
  - 'A tag'
  - 'A second tag'
  - 'An nth tag'
CatalogContent:
  - 'learn-example-course'
  - 'path/example-path'
---